.github/workflows/ci.yml

```diff
@@ -275,11 +259,14 @@ jobs:
           find src -name '*.re' | xargs touch
           cmake --build ${{ matrix.build-dir }} --config ${{ matrix.build-type }} --verbose
 
-      - name: Standard Tests
-        if: startsWith(runner.os, 'Windows') == false &&  matrix.skeleton == false
-        run: cmake --build ${{ matrix.build-dir }} --target check
-
-      - name: Skeleton Validation
-        if: startsWith(runner.os, 'Windows') == false && matrix.skeleton == true
+      - name: Test
+        if: startsWith(runner.os, 'Windows') == false
         working-directory: ${{ matrix.build-dir }}
-        run: ./run_tests.sh --skeleton
+        run: |
+          # main test suite
+          cmake --build . --target check
```

Comment:
It should be:

```suggestion
          cmake --build ${{ matrix.build-dir }} --target check
```

Reply:
But I have added `working-directory: ${{ matrix.build-dir }}`, doesn't it mean that `.` is `${{ matrix.build-dir }}` ?

Reply:
Oh! You're right!

Reply:
Also, the tests do run.